mpas_analysis/sea_ice/timeseries.py

```diff
@@ -238,39 +269,41 @@ def seaice_timeseries(config):
                                          title_font_size=title_font_size)
             else:
                 # we will combine north and south onto a single graph
-                figname = "%s/%s.%s.png" % (plots_dir,casename,varname)
-                title = "%s, NH (r), SH (k)\n%s" % (plot_title,casename)
+                figname = "{}/{}.{}.png".format(plots_dir, casename, varname)
+                title = "{}, NH (r), SH (k)\n{}".format(plot_title, casename)
                 timeseries_analysis_plot(config, [var_nh, var_sh], N_movavg,
                                          title, xlabel, units, figname,
-                                         lineStyles=['r-','k-'],
+                                         lineStyles=['r-', 'k-'],
                                          lineWidths=[1.2, 1.2],
                                          title_font_size=title_font_size)
 
         elif varname == "iceThickCell":
 
-            figname = "%s/%s.%s.png" % (plots_dir,casename,varname)
-            title = "%s NH (r), SH (k)\n%s" % (plot_title,casename)
-            timeseries_analysis_plot(config, [var_nh,var_sh], N_movavg, title,
-                                      xlabel, units, figname,
-                                      lineStyles=['r-','k-'],
-                                      lineWidths=[1.2, 1.2],
-                                      title_font_size=title_font_size)
+            figname = "{}/{}.{}.png".format(plots_dir, casename, varname)
+            title = "{} NH (r), SH (k)\n{}".format(plot_title, casename)
+            timeseries_analysis_plot(config, [var_nh, var_sh], N_movavg, title,
+                                     xlabel, units, figname,
+                                     lineStyles=['r-', 'k-'],
+                                     lineWidths=[1.2, 1.2],
+                                     title_font_size=title_font_size)
 
         else:
-            raise SystemExit("varname variable %s not supported for plotting" % varname)
+            raise ValueError(
+                "varname variable {} not supported for plotting".format(
+                    varname))
```

Comment:
Yeah I don't think we want *any* calls to `SystemExit` anywhere in the code.  It's not the right thing to be raising.